provider/helpers.go

```diff
@@ -55,3 +55,21 @@ func shouldPatchString(state, plan types.String) (value string, ok bool) {
 	}
 	return
 }
+
+// shouldPatchString changed checks if the change between the given state and plan requires sending
+// a patch request to the server. If it does it returns the value to add to the patch.
+func shouldPatchBool(state, plan types.Bool) (value bool, ok bool) {
```

Comment:
Where is it used from?

Reply:
I thought to use it for patching autoscaling but I removed it 
thanks